api.web.yaml

```diff
@@ -334,6 +335,9 @@ components:
               - admin
               - base
               - approved-researcher
+        isUclStaff:
+          type: boolean
+          description: Whether the user is a valid UCL staff member
```

Comment:
This is one idea for how to add further checks on the frontend, and might be useful for more upcoming features. But we can think more about this if needed.